File: RxUI/src/main/java/com/tamsiree/rxui/view/colorpicker/renderer/ColorWheelRenderer.java
package com.tamsiree.rxui.view.colorpicker.renderer;


import com.tamsiree.rxui.view.colorpicker.ColorCircle;

import java.util.List;

/**
 * @author tamsiree
 * @date 2018/6/11 11:36:40 整合修改
 */
public interface ColorWheelRenderer {
    float GAP_PERCENTAGE = 0.025f;

    void draw();

    ColorWheelRenderOption getRenderOption();

    void initWith(ColorWheelRenderOption colorWheelRenderOption);

    List<ColorCircle> getColorCircleList();
}

File: RxUI/src/main/java/com/tamsiree/rxui/view/colorpicker/renderer/ColorWheelRenderer.kt
package com.tamsiree.rxui.view.colorpicker.renderer

import com.tamsiree.rxui.view.colorpicker.ColorCircle

/**
 * @author tamsiree
 * @date 2018/6/11 11:36:40 整合修改
 */
interface ColorWheelRenderer {
    fun draw()
    val renderOption: ColorWheelRenderOption
    fun initWith(colorWheelRenderOption: ColorWheelRenderOption)
    val colorCircleList: List<ColorCircle>?

    companion object {
        const val GAP_PERCENTAGE = 0.025f
    }
}